src/styles/style.scss

```diff
@@ -23,6 +26,16 @@ input:-webkit-autofill:active {
     transition: background-color 5000s ease-in-out 0s;
 }
 
+::-webkit-scrollbar {
+    display: none;
+}
+
+/* ползунок скроллбара */
```

Comment:
remove comment

Reply:
Tsis is on old commits, everything has already been fixed